doc/classes/ProjectSettings.xml

```diff
@@ -235,6 +235,16 @@
 		</method>
 	</methods>
 	<members>
+		<member name="accessibility/accessibility/accessibility_support" type="int" setter="" getter="" default="0">
```

Comment:
The `accessibility/accessibility/` seems a bit weird.
Maybe it should be `accessibility/general/`?

Reply:
Changed to `accessibility/general/`.